src/CommunityToolkit/Xamarin.CommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="MSBuild.Sdk.Extras/2.1.2">
   <PropertyGroup>
     <TargetFrameworks>netstandard1.0;netstandard2.0;netstandard2.1;Xamarin.iOS10;MonoAndroid10.0;Xamarin.TVOS10;Xamarin.WatchOS10;Xamarin.Mac20;tizen40</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(TargetFrameworks);uap10.0.17763;netcoreapp3.1;net472;net471</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(TargetFrameworks);uap10.0.19041;netcoreapp3.1;net472;net471</TargetFrameworks>
```

Comment:
Do we need to upgrade this version? I mean is this better than using the WinUI (I remember that we talked about it on the discord channel)?

Reply:
I needed to update to 19041 to get the Flyout to render with a transparent background. Prior to 18362 the Flyout Shadow was not configurable which means there would be a weird shadow if you set the Color to transparent. I am not sure the best way to handle this as I added a pre-processor directive `#if` to only use the property if the windows version is >= 18362. 

PopupRenderer.uwp.cs:168-171
```c#
#if UWP_18362
    if (Element.Color == Color.Transparent)
        flyoutStyle.Setters.Add(new Windows.UI.Xaml.Setter(FlyoutPresenter.IsDefaultShadowEnabledProperty, false));
#endif
```

In the UWP Sample App it is using 18362 as the target but 17763 as the min version. I think this is 'okay' but if anyone knows a better way to manage this I am all ears.

![image](https://user-images.githubusercontent.com/17751436/101367191-e50a8b00-3873-11eb-9cba-94698d7d1d83.png)


### WinUI Investigation
While implementing this I tried using WinUI instead of the UWP library as the `Flyout` control is available in WinUI 3. Xamarin.Forms Shell takes a dependency on WinUI 2 and `Flyout` isn't available in WinUI 2. It would create a management nightmare for XCT to include WinUI 3 and Xamarin.Forms to include WinUI 2. 

WinUI 3 is still in preview and I don't think it makes sense to do any push for supporting it in Xamarin.Forms 5 at this point. Perhaps once Xamarin.Forms 5 is shipped and WinUI 3 is released we could revisit this idea

Reply:
Said that, is there a problem to have a mismatch version with the XF or apps made with XF? If don't have any compatibility problem I'm ok with that.
Because I can remember that apps can have a problem with the supported version, see this #624.